pages/dashboard.js

```diff
@@ -114,50 +123,31 @@ const Dashboard = () => {
 
   return (
     <PageWrapper title='Dashboard'>
-      <Section backgroundColor='lightRock'>
-        {showWelcomeMessage && (
-          <Flex
-            position='relative'
-            maxW={{ md: '60rem' }}
-            color='ocean'
-            backgroundColor='lightPuddle'
-            direction={{ base: 'column', md: 'row' }}
-            align='center'
-            justify='center'
-            padding={{ base: '1rem 1.5rem', md: '1.5rem 3rem' }}
-            margin='0 auto 3rem'
-            fontWeight='500'
-          >
-            <Icon
-              name='hooray'
-              size={{ base: '2rem', md: '3rem' }}
-              marginRight={{ base: 0, md: '1.5rem' }}
-              marginBottom={{ base: '1.5rem', md: 0 }}
-            />
+      {showWelcomeMessage && (
```

Comment:
this banner shouldn't show if the other banner is showing; their wording conflicts and 2 banners at once is overwhelming

Reply:
Hmmmm unsure how that logic should exist for which banner to show and when. I don’t mind two banners but if we do, when do we show the welcome banner if they installed later?

What do you suggest